compiling after switching to pe 1.0.0 prerelease and installing everything for it brings up this error

### Describe your bug here.

i wanted to switch to pe source code 1.0.0 prerelease from 0.7.3 and after installing the missing libraries from the bat file and fixing sscript again i get this error message

### Command Prompt/Terminal logs (if existing)

```bash
Error: __boot__.cpp
include\backend/DiscordClient.h(11): fatal error C1083: Non è possibile aprire il file inclusione: 'discord_rpc.h': No such file or directory
```


### Are you modding a build from source or with Lua?

Source

### What is your build target?

Windows

### Did you edit anything in this build? If so, mention or summarize your changes.

no, i didnt edit anything yet

--- a/Project.xml
+++ b/Project.xml
@@ -112,7 +112,7 @@
 	<haxelib name="linc_luajit" if="LUA_ALLOWED"/>
 	<haxelib name="hscript-iris" if="HSCRIPT_ALLOWED" version="1.0.2"/>
 	<haxelib name="hxvlc" if="VIDEOS_ALLOWED"/>
-	<haxelib name="hxdiscord_rpc" if="DISCORD_ALLOWED"/>
+	<haxelib name="hxdiscord_rpc" version="1.2.1" if="DISCORD_ALLOWED"/>
 	<haxelib name="flxanimate"/>
 
 	<!-- Disable Discord IO Thread -->

--- a/setup/unix.sh
+++ b/setup/unix.sh
@@ -12,9 +12,9 @@ haxelib install flixel-addons 3.2.2
 haxelib install flixel-tools 1.5.1	
 haxelib install hscript-iris 1.0.2
 haxelib install tjson 1.4.0
+haxelib install hxdiscord_rpc 1.2.1
 haxelib git flxanimate https://github.com/Dot-Stuff/flxanimate 768740a56b26aa0c072720e0d1236b94afe68e3e
 haxelib git linc_luajit https://github.com/superpowers04/linc_luajit 633fcc051399afed6781dd60cbf30ed8c3fe2c5a
-haxelib git hxdiscord_rpc https://github.com/MAJigsaw77/hxdiscord_rpc 3538a1c2bb07b04208cd014220207f8173acdb21
 haxelib git hxvlc https://github.com/MAJigsaw77/hxvlc 70e7f5f3e76d526ac6fb8f0e6665efe7dfda589d
 haxelib git funkin.vis https://github.com/FunkinCrew/funkVis d5361037efa3a02c4ab20b5bd14ca11e7d00f519
 haxelib git grig.audio https://gitlab.com/haxe-grig/grig.audio.git cbf91e2180fd2e374924fe74844086aab7891666

--- a/setup/unix.sh
+++ b/setup/unix.sh
@@ -14,7 +14,7 @@ haxelib install flixel-addons 3.2.2
 haxelib install flixel-tools 1.5.1	
 haxelib install hscript-iris 1.0.2
 haxelib install tjson 1.4.0
-haxelib install hxdiscord_rpc 1.2.1
+haxelib install hxdiscord_rpc 1.2.4
 haxelib git flxanimate https://github.com/Dot-Stuff/flxanimate 768740a56b26aa0c072720e0d1236b94afe68e3e
 haxelib git linc_luajit https://github.com/superpowers04/linc_luajit 633fcc051399afed6781dd60cbf30ed8c3fe2c5a
 haxelib git hxvlc https://github.com/MAJigsaw77/hxvlc 70e7f5f3e76d526ac6fb8f0e6665efe7dfda589d

--- a/setup/windows.bat
+++ b/setup/windows.bat
@@ -11,9 +11,9 @@ haxelib install flixel-addons 3.2.2
 haxelib install flixel-tools 1.5.1
 haxelib install hscript-iris 1.0.2
 haxelib install tjson 1.4.0
+haxelib install hxdiscord_rpc 1.2.1
 haxelib git flxanimate https://github.com/Dot-Stuff/flxanimate 768740a56b26aa0c072720e0d1236b94afe68e3e
 haxelib git linc_luajit https://github.com/superpowers04/linc_luajit 633fcc051399afed6781dd60cbf30ed8c3fe2c5a
-haxelib git hxdiscord_rpc https://github.com/MAJigsaw77/hxdiscord_rpc 3538a1c2bb07b04208cd014220207f8173acdb21
 haxelib git hxvlc https://github.com/MAJigsaw77/hxvlc 70e7f5f3e76d526ac6fb8f0e6665efe7dfda589d
 haxelib git funkin.vis https://github.com/FunkinCrew/funkVis d5361037efa3a02c4ab20b5bd14ca11e7d00f519
 haxelib git grig.audio https://gitlab.com/haxe-grig/grig.audio.git cbf91e2180fd2e374924fe74844086aab7891666

--- a/setup/windows.bat
+++ b/setup/windows.bat
@@ -11,7 +11,7 @@ haxelib install flixel-addons 3.2.2
 haxelib install flixel-tools 1.5.1
 haxelib install hscript-iris 1.0.2
 haxelib install tjson 1.4.0
-haxelib install hxdiscord_rpc 1.2.1
+haxelib install hxdiscord_rpc 1.2.4
 haxelib git flxanimate https://github.com/Dot-Stuff/flxanimate 768740a56b26aa0c072720e0d1236b94afe68e3e
 haxelib git linc_luajit https://github.com/superpowers04/linc_luajit 633fcc051399afed6781dd60cbf30ed8c3fe2c5a
 haxelib git hxvlc https://github.com/MAJigsaw77/hxvlc 70e7f5f3e76d526ac6fb8f0e6665efe7dfda589d

